lib/wallet/test/unit/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -351,7 +341,8 @@ import qualified Cardano.Wallet.Primitive.Types.Coin as Coin
 import qualified Cardano.Wallet.Primitive.Types.TokenBundle as TokenBundle
 import qualified Cardano.Wallet.Primitive.Types.Tx.TxOut as TxOut
 import qualified Cardano.Wallet.Primitive.Types.Tx.TxOut.Gen as TxOutGen
-import qualified Cardano.Wallet.Shelley.Compatibility as Compatibility
+import qualified Cardano.Wallet.Shelley.Compatibility as Convert
```

Comment:
I don't think we should depend on the huge `Shelley.Compatibility` module.

What would you think of doing something like https://github.com/cardano-foundation/cardano-wallet/pull/4154 ? Of course, we'd hopefully reduce the number of types and the need for conversions in the future.

Reply:
> I don't think we should depend on the huge Shelley.Compatibility module.

I agree! I hope we can remove it. (And I see you've made an additional PR that does so -- thanks!)

Though this PR was just intended to rename the namespace and do nothing else. (It doesn't introduce any new dependencies.)

Reply:
@Anviking I've cherry-picked your commit from #4154 into this PR.